admin.go

```diff
@@ -380,3 +417,87 @@ func (ca *clusterAdmin) DeleteACL(filter AclFilter, validateOnly bool) ([]Matchi
 	}
 	return mAcls, nil
 }
+
+func (ca *clusterAdmin) DescribeConsumerGroup(group string) (*GroupDescription, error) {
```

Comment:
I wonder if it works better as `DescribeConsumerGroups(groups []string)`, coz controller takes a slice of string anyway, and that way if you only want it for one group, you can only pass that too?

Reply:
i agree. i now improved it to accept a slice of group names. in addition, the logic is now quite smart to first build a map of brokers -> []string groups, so each group coordinator is contacted only once. in a trivial test / cluster with just 8 groups this saves me a second (50% savings)..which will have much a bigger impact on larger clusters.